strawberry/http/sync_base_view.py

```diff
@@ -112,11 +113,15 @@ def execute_operation(
 
         assert self.schema
 
+        context_wrapper = ContextWrapper(
+            context=context, extensions=request_data.extensions
+        )
```

Comment:
is this context wrapper only to pass request data?

Reply:
yes exactly. Since the `context` data comes from `get_context` and that can be any type of object the user decides to return, this felt like a safe way to attach more data

Reply:
I'll try to review and merge this this week 😊 thanks for the patience!